OpenRIAServices/OpenRiaServices
# Run AspNetCore E2E tests with NET8 on server

.NET8
  * Build with dotnet8 sdk
  * Run AspNetCore E2E tests with .NET8 on server
  * Run AspNetCore tests on .NET8

Test improvements
* improve test failure message from GetReferenceAssemblies
* Verify AspNetCore webserver is started before executing e2e tests
* Fix invalid handle from CrossProcessLoggingTests when debugger is attached

Build improvements
* Set MSBUILDDEBUGPATH to allow troubleshooting of build failures
* Stop installing dotnet (UseDotNet) for slightly faster build
* remove appveyor file
* Use msbuild directly in codeql
 * enable test diagnostics
